agent/util-scripts/test-bin/rpm

```diff
@@ -19,8 +19,13 @@ fi
 
 _rpm="${3}"
 if [[ "${_rpm}" == "pbench-sysstat" ]]; then
-    echo "12.0.3-1"
-    _rc=0
+    if [[ ${_PBENCH_UNIT_TEST_NAME} == "test-61" ]]; then
```

Comment:
While I'm not thrilled about this, maybe it's not worth working out a better hack.